resources/sass/app.scss

```diff
@@ -5,8 +5,20 @@
 @import '~@wmde/wikit-tokens/dist/_variables.scss';
 @import './_typography.scss';
 
+html {
+    height: 100%;
+}
+
 body {
     @include body-text;
+    height: 100%;
+}
+
+#app {
+    box-sizing: border-box;
+    min-height: 100%;
+    display: flex;
+    flex-direction: column;
```

Comment:
That’s great and works fantastically. Though, since these rules are more thematically related to the layout, I'd say we should move them into `Layout.vue` under the `.website` selector (it's actually the same element as `#app` after Vue is initialized).

Reply:
Ah no, I'm suggesting you move these rules there since they mostly relate to the html that is being produced by `Layout.vue`. In this file I would only add rules that relate to `html` and `body` or very general rules that are agnostic to the content of the app (such as reset rules for example). Although I do see other rules in this sass file that I think should move to more appropriate places, but that's for another task, not this one.